src/App.js

```diff
@@ -1,23 +1,24 @@
-import logo from './logo.svg';
 import './App.css';
+import { Route, Routes } from 'react-router-dom';
+import Navbar from './components/Navbar/Navbar';
+import HomeScreen from './screens/HomeScreen/HomeScreen';
+import DetailsScreen from './screens/DetailsScreen/DetailsScreen';
+import BeerScreen from './screens/BeerScreen/BeerScreen';
+import NewBeerScreen from './screens/NewBeerScreen/NewBeerScreen';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Navbar />
+        <div className='Routes'>
+          <Routes>
+            <Route path="/" element={<HomeScreen />} />
+            <Route path='/beers' element={<BeerScreen />} />
+            <Route path="/random-beer" element={<DetailsScreen />} />
+            <Route path='/beers/:id' element={<DetailsScreen />} />
+            <Route path='/beers/new' element={<NewBeerScreen />} />
+          </Routes>
```

Comment:
aquí te faltaría redirigir al usuario a la home cada vez que navegue a una ruta inexistente:
`<Route path='*' element={<Navigate to='/' />} />`